service/rate.go

```diff
@@ -64,6 +64,11 @@ func (h *Service) HandleLinkMessage(req LinksRequest) (string, error) {
 	h.currentlyWorkingOrders.Store(groupID.ID, nil)
 	defer h.currentlyWorkingOrders.Delete(groupID.ID)
 
+	err := h.eventNotification.AddReaction(req.Channel, req.MessageID, h.cfg.JoinedOrderEmoji)
+	if err != nil {
+		return "", fmt.Errorf("adding reaction to link message: %w", err)
```

Comment:
Slack sends link events even for channels that the bot is not part of, and then when trying to send a message, it gets an error. I had a bug when it first joined the wolt order and then sent the message, and just then, the bot discovered that it is not part of the channel and it can't send messages. To solve that, I changed it so it first sends a message and if it gets an error, it returns `errWontJoin` and won't join the wolt order and not log the error (because it is not really an error). So It's better to return here `errWontJoin`.
```suggestion
		return "", errWontJoin
```